### ISSUE TITLE
Content type header for latest chewy

### ISSUE THREAD

**Issue by FactoryGirl** [toptal/chewy#648]
Guyz, on latest chewy and elasticsearch 6.4 I get error
`[406] {"error":"Content-Type header [application/x-www-form-urlencoded] is not supported","status":406}):
`
Seems like 6.4 requires content type header set.  But there is no possibility to set it in chewy. How come?

**Comment by pyromaniac** [comment]
I think you can set it in the client

**Comment by FactoryGirl** [comment]
@pyromaniac how? and where?

**Comment by FactoryGirl** [comment]
Official documentation says you must send this header on request https://www.elastic.co/blog/strict-content-type-checking-for-elasticsearch-rest-requests

**Comment by pyromaniac** [comment]
Yeah, you are right, but I propose to do it manually for now as a workaround: https://github.com/toptal/chewy#aws-elastic-search - here is an example.
